### ISSUE TITLE
Hey

### ISSUE THREAD

**Issue by mzzovko** [BadCityGame/BadCity#8]
I've tried the game and when I join the server my game looks like this:
http://imgur.com/a/zbArx

My ping is really high though, like 200-250

**Comment by DVLP** [comment]
Are you using the latest version of Firefox?

**Comment by mzzovko** [comment]
Yes, Also I tried other server with lower ping, and character model rendered okay. I was able to move and shoot etc. But there was no other models, only floor plane with texture on it and that's it.

**Comment by DVLP** [comment]
That's just an empty map. All maps are created by players and many were created without any content(testing only?). Now this behaviour is blocked and there must be some objects on the map to be able to publish it. We'll remove empty maps today

**Comment by DVLP** [comment]
All empty maps have been removed, but the rest will need quality control. You can try tomorrow when there will be more players
